test/server/servers/SockJSServer.test.js

```diff
@@ -84,9 +84,20 @@ describe('SockJSServer', () => {
         done();
       }, 3000);
     });
+
+    it('should not throw an exception when connection is null', (done) => {
+      expect(() => {
+        socketServer.onConnection(() => {});
+        socketServer.socket.emit('connection', null);
+      }).not.toThrow();
+
+      setTimeout(() => {
```

Comment:
This timeout is not needed, remove this and remove the `done` parameter from the function.

Reply:
* Reason for the `timeout`
It won't pass in the build machine only for Windows node-8 and Windows node-6.
![image](https://user-images.githubusercontent.com/5212215/63313860-bcdfcd00-c2ba-11e9-8bb0-89565b9bca2e.png)
See https://dev.azure.com/webpack/webpack-dev-server/_build/results?buildId=5531

Reply:
That is caused by a stability issue with our e2e tests. You can ignore it, your changes are not causing it (you can see that your test ran and passed after the problematic test), and your changes can still be merged if CI is failing for this reason. The `timeout` can still be removed.

Reply:
Thanks. I am going to make the change

Reply:
Yes, it was the stability issue. Right now, all are passed. Thanks